package.json

```
@@ -22,15 +22,15 @@
   },
   "scripts": {
     "pre-reqs": "npm install -g typescript prettier",
-    "clean": "rm -rf packages/*/tsconfig.tsbuildinfo && rm -rf coverage test-report && lerna clean --yes && rm -rf node_modules && yarn cache clean",
+    "clean": "lerna run clean --scope @shapeshiftoss/* && rm -rf coverage test-report node_modules && yarn cache clean",
     "lint:ts": "tsc --noEmit",
     "format": "prettier .",
-    "yarn": "yarn",
     "build": "yarn tsc --build",
     "dev:all": "yarn tsc --build --watch",
     "now-build": "lerna run build --scope @shapeshiftoss/hdwallet-core && lerna run build --scope hdwallet-sandbox --ignore @shapeshiftoss/hdwallet-keepkey-nodehid --include-filtered-dependencies && mv examples/sandbox/public .",
     "docker:run:emulator": "docker run -p 5000:5000 kktech/kkemu:latest",
     "docker:stop:run:emulator": "docker stop $(docker ps -a -q --filter ancestor=kktech/kkemu:latest --format=\"{{.ID}}\") && docker run -p 5000:5000 kktech/kkemu:latest",
+    "sandbox": "cd examples/sandbox && yarn dev",
```

so this actually used to `dev:sandbox` I had accidentally removed it a little while back and it is still referenced in the readme. can we use this instead?
```suggestion
    "dev:sandbox": "lerna run dev --scope hdwallet-sandbox --parallel --include-filtered-dependencies",
```